.pre-commit-config.yaml

```diff
@@ -553,7 +553,19 @@ repos:
           ^airflow/example_dags/example_sensors.py|
           ^airflow/example_dags/example_sensors.py|
           ^airflow/example_dags/example_sensors.py|
-          ^airflow/example_dags/example_time_delta_sensor_async.py
+          ^airflow/example_dags/example_time_delta_sensor_async.py|
```

Comment:
For this change - if we basically add all example DAGs into the exclude list - the check has almost no effect.

Can you re-work the check such that no provider except standard provider dependencies are accepted?

Reply:
Yes, agree. will check.

Reply:
updated to allow only imports from standard provider :)

Reply:
Hope the updated check is fine?